### PR TITLE
WebAPI MongoDB: Clarify steps

### LINKED ISSUE #30010: It Does Not Work
I followed all the steps but when I got to "Test the web API" I just got nothing, a blank array for /Books and "not found" when I tried a specific book ID. 

The DB connection info appears correct and I can see my Bookstore database and Books table in Compass and the sample data is exactly as suggested. 

So how do you troubleshoot when there's no real error message that would help me figure out why the API returns a blank array instead of a list of my books? 

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: 5240866d-d0a1-f0dd-7b2c-203e1d69bc4b
* Version Independent ID: 83a5cb3b-23bc-4f0c-cd1c-1d1ffeb1131a
* Content: [Create a web API with ASP.NET Core and MongoDB](https://learn.microsoft.com/en-us/aspnet/core/tutorials/first-mongo-app?view=aspnetcore-7.0&tabs=visual-studio)
* Content Source: [aspnetcore/tutorials/first-mongo-app.md](https://github.com/dotnet/AspNetCore.Docs/blob/main/aspnetcore/tutorials/first-mongo-app.md)
* Product: **aspnet-core**
* Technology: **aspnetcore-tutorials**
* GitHub Login: @wadepickett
* Microsoft Alias: **wpickett**

### PR DESCRIPTION
**Pull request by wadepickett**
Fixes #30010 

Tested on VS and macOS scenarios.

- The original path setting instruction was confusing and not organized to clearly separate Windows from macOS/Linux. Rewrote to clarify.
- The instruction for the Open API test pages did not include selecting **Try it out** > **Execute**.  Added that.
- Updated VS for Mac instruction to remove Preview settings steps.

---

#### Internal previews

| 📄 File | 🔗 Preview link |
|:--|:--|
| [aspnetcore/tutorials/first-mongo-app.md](https://github.com/dotnet/AspNetCore.Docs/blob/7bb4ebeb8b9699b8b171945fc2801d4045d87206/aspnetcore/tutorials/first-mongo-app.md) | [Create a web API with ASP.NET Core and MongoDB](https://review.learn.microsoft.com/en-us/aspnet/core/tutorials/first-mongo-app?branch=pr-en-us-32353) |

### CODE REVIEW

**Comment by wadepickett** [comment]
Forcing rebuild, build did not finish.

File: aspnetcore/tutorials/first-mongo-app.md [hunk 1]
```diff
@@ -49,11 +49,18 @@ In this tutorial, you learn how to:
...
 
-Enable MongoDB and Mongo DB Shell access from anywhere on the development machine:
 
-1. On Windows, MongoDB is installed at *C:\\Program Files\\MongoDB* by default. Add *C:\\Program Files\\MongoDB\\Server\\\<version_number>\\bin* to the `PATH` environment variable.
-1. Download the MongoDB Shell and choose a directory to extract it to. Add the resulting path for `mongosh.exe` to the `PATH` environment variable.
-1. Choose a directory on the development machine for storing the data. For example, *C:\\BooksData* on Windows. Create the directory if it doesn't exist. The mongo Shell doesn't create new directories.
+Enable MongoDB and Mongo DB Shell access from anywhere on the development machine (Windows/Linux/macOS):
+
+1. Download and Install MongoDB Shell:
+   * macOS/Linux: Choose a directory to extract the MongoDB Shell to. Add the resulting path for `mongosh` to the `PATH` environment variable.
```
Comment (tdykstra) [inline aspnetcore/tutorials/first-mongo-app.md:56]: One spot has Mongo DB Shell, the other has MongoDB Shell

**Review by tdykstra — APPROVED** [review]
I did review it earlier but neglected to post my review 😄

File: aspnetcore/tutorials/first-mongo-app.md (hunk 1 above)
Comment (wadepickett) [reply aspnetcore/tutorials/first-mongo-app.md:56]: Will fix.  Should be MongoDB Shell.

File: aspnetcore/tutorials/first-mongo-app.md [hunk 2]
```diff
@@ -49,11 +49,18 @@ In this tutorial, you learn how to:
 
 ## Configure MongoDB
 
-Enable MongoDB and Mongo DB Shell access from anywhere on the development machine:
 
-1. On Windows, MongoDB is installed at *C:\\Program Files\\MongoDB* by default. Add *C:\\Program Files\\MongoDB\\Server\\\<version_number>\\bin* to the `PATH` environment variable.
-1. Download the MongoDB Shell and choose a directory to extract it to. Add the resulting path for `mongosh.exe` to the `PATH` environment variable.
-1. Choose a directory on the development machine for storing the data. For example, *C:\\BooksData* on Windows. Create the directory if it doesn't exist. The mongo Shell doesn't create new directories.
+Enable MongoDB and Mongo DB Shell access from anywhere on the development machine (Windows/Linux/macOS):
```
Comment (wadepickett) [inline aspnetcore/tutorials/first-mongo-app.md:53]:
```suggestion
Enable MongoDB and MongoDB Shell access from anywhere on the development machine (Windows/Linux/macOS):
```

File: aspnetcore/tutorials/first-mongo-app.md [hunk 3]
```diff
@@ -49,11 +49,18 @@ In this tutorial, you learn how to:
...
+
+1. Download and Install MongoDB Shell:
+   * macOS/Linux: Choose a directory to extract the MongoDB Shell to. Add the resulting path for `mongosh` to the `PATH` environment variable.
+   * Windows: MongoDB Shell (mongosh.exe) is installed at *C:\Users\<user>\AppData\Local\Programs\mongosh*. Add the resulting path for `mongosh.exe` to the `PATH` environment variable.
+2. Download and Install MongoDB:
+   * macOS/Linux: Verify the directory that MongoDB was installed at, usually in */usr/local/mongodb*. Add the resulting path for `mongodb` to the `PATH` environment variable.
+   * Windows: MongoDB is installed at *C:\\Program Files\MongoDB* by default. Add *C:\\Program Files\\MongoDB\\Server\\\<version_number>\\bin* to the `PATH` environment variable.
+1. Choose a Data Storage Directory: Select a directory on your development machine for storing data. Create the directory if it doesn't exist. The mongo Shell doesn't create new directories:
+   * macOS/Linux: For example, `/usr/local/var/mongodb`.
+   * Windows: For example, `C:\\BooksData`.
```
[the remark this answers is not in the document]
Comment (wadepickett) [reply aspnetcore/tutorials/first-mongo-app.md:63]: Good question. On Linux for some distributions and MongoDB installations a common default path used is `/usr/local/var/mongodb` so it gets suggested a lot on stackoverflow/tutorials, so I did not break from that, although I could set it to anything.  For Windows you can put it anywhere.

**Pull request merged**